railties/lib/rails/application/configuration.rb

```diff
@@ -336,6 +336,7 @@ def load_defaults(target_version)
           if respond_to?(:active_record)
             active_record.postgresql_adapter_decode_dates = true
             active_record.validate_migration_timestamps = true
+            active_record.attributes_for_inspect = :all
```

Comment:
```suggestion
            active_record.attributes_for_inspect = [:id]
```

This one should be the _new_ default that apps can opt into when they `load_defaults 7.2` after upgrading.

(The line may have to be more complicated than this... maybe 
```[:id] unless Rails.env.local?``` or something similar?)